### ISSUE TITLE
Feedback from peers

### ISSUE THREAD

**Issue by EvanYathon** [UBC-MDS/Capital_Bikeshare_Analysis#37]
- [x] Include train accuracy on max depth plot. https://github.com/UBC-MDS/Capital_Bikeshare_Analysis/commit/b88f13eea85c36f141eb03c17c8a7177ffecc911
- [x] Explicitly mention test accuracy value to elaborate more about hyperparameter testing.
- [x] Can we use casual and membership as features?  Talk about our choice not to in the report.  https://github.com/UBC-MDS/Capital_Bikeshare_Analysis/commit/4084c9439d776e2a354d885f4f18c6b5515a7c72

- [ ]   over plotting for figure 1
- [x]    workflow and five script need to be included README https://github.com/UBC-MDS/Capital_Bikeshare_Analysis/commit/0ddf0c13fd7bbd46c92ba87de5e45cb09bd56cf5
- [x]    double check with cross validation accuracy with the final model, use approximation error instead https://github.com/UBC-MDS/Capital_Bikeshare_Analysis/commit/b88f13eea85c36f141eb03c17c8a7177ffecc911

**Comment by EvanYathon** [comment]
@Mikeymice go ahead and add your feedback too and we can choose a couple to address for this week's milestone

**Comment by Mikeymice** [comment]
sounds good. i already added some. will work on them this evening :)

**Comment by EvanYathon** [comment]
Me too, working on 561 this afternoon.  I can take care of the casual/members part and include it in the report.  If we do just one other thing such as approximation error on the graph I think we would be good!  Thoughts?

**Comment by Mikeymice** [comment]
sounds good !

**Comment by Mikeymice** [comment]
for the suggestion of motion test accuracy, i thought we did in the report ?

**Comment by Mikeymice** [comment]
for the workflow i am thinking of include this into our README 
![flowchart](https://user-images.githubusercontent.com/15175882/49324041-caf67f00-f4da-11e8-8077-9115283f3296.png)

**Comment by EvanYathon** [comment]
> for the suggestion of motion test accuracy, i thought we did in the report ?

Oh you did!  Sorry, I was looking in the wrong place in our report.

With regards to that workflow I think it looks awesome!  Let's do it

**Comment by EvanYathon** [comment]
I'll close the issue now, I think I'll leave the overplotting out of it as I personally actually prefer the way we have it now.  I think it shows the variation in day to day ridership and doesn't mask any data while still being able to show overall trends.

**Comment by Mikeymice** [comment]
Sounds good
